nap/docker/ckan/ckan-plugins/ckanext-napote_theme/ckanext/napote_theme/templates/transport_service_view.html

```diff
@@ -0,0 +1,24 @@
+<link rel="stylesheet" href="https://unpkg.com/leaflet@1.2.0/dist/leaflet.css"
+      integrity="sha512-M2wvCLH6DSRazYeZRIm1JnYyh22purTM+FDB5CsyxtQJYeKq83arPe5wgbNmcFXGqiSH2XR8dT/fJISVA1r/zQ=="
+      crossorigin=""/>
+
+<style id="_stylefy-constant-styles_"></style>
+<style id="_stylefy-styles_"></style>
+
+<style>
```

Comment:
I would keep styles out of templates. This is good for now, but if there will be more style rules, it would be better to place them into css-directory of the plugin.